ui/components/application/View.tsx

```diff
@@ -0,0 +1,359 @@
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.eventsPerWeek")}
+                value={applicationEvent.eventsPerWeek}
+              />
+              <div />
+            </TwoColumnContainer>
+
+            <FormSubHeading>
+              {t("application:Page1.spacesSubHeading")}
+            </FormSubHeading>
+            <UnitList>
+              {sortBy(applicationEvent.eventReservationUnits, "priority").map(
+                (reservationUnit, index) => {
+                  const resUnit = get(reservationUnits, [
+                    reservationUnit.reservationUnitId,
+                  ]);
+                  return (
+                    <UnitName key={reservationUnit.reservationUnitId}>
+                      <div>{index + 1}</div>
+                      <div>
+                        {trim(
+                          `${getOldReservationUnitName(
+                            reservationUnit.reservationUnitDetails
+                          )}${
+                            resUnit &&
+                            `, ${localizedValue(
+                              get(resUnit, "unit.name"),
+                              i18n.language
+                            )}`
+                          }`,
+                          ","
+                        )}
+                      </div>
+                    </UnitName>
+                  );
+                }
+              )}
+            </UnitList>
+            <FormSubHeading>
+              {t("application:preview.applicationEventSchedules")}
+            </FormSubHeading>
+            <TimePreviewContainer data-testid={`time-selector__preview-${i}`}>
+              <TimePreview
+                applicationEventSchedules={[
+                  summaryDataPrimary,
+                  summaryDataSecondary,
+                ]}
+              />
+            </TimePreviewContainer>
+          </Accordion>
+        );
+      })}
+      <FormSubHeading>{t("reservationUnit:termsOfUse")}</FormSubHeading>
+      <Terms tabIndex={0}>{getTranslation(tos1, "text")}</Terms>
+      <FormSubHeading>
+        {t("application:preview.reservationUnitTerms")}
+      </FormSubHeading>
+      <Terms tabIndex={0}>{getTranslation(tos2, "text")}</Terms>
+      <CheckboxContainer>
+        <Checkbox
+          id="preview.acceptTermsOfUse"
+          checked={acceptTermsOfUse}
+          onChange={(e) => setAcceptTermsOfUse(e.target.checked)}
+          label={t("application:preview.userAcceptsTerms")}
+          disabled
+        />
+      </CheckboxContainer>
+      <StyledNotification
+        label={t("application:preview.notification.processing")}
+      >
+        {t("application:preview.notification.body")}
+      </StyledNotification>
+      <ButtonContainer>
+        <BlackButton
+          variant="secondary"
+          onClick={() => router.push(`${applicationsUrl}`)}
```

Comment:
Why router.push? router.back seems more approapriate, or router.replace. Or is there a case where we want user to hit "Cancel" and then "back" on the browser to get them back to the form.